### ISSUE TITLE
Console.log crashes fs-jetpack

### ISSUE THREAD

**Issue by black-snow** [szwacz/fs-jetpack#29]
`console.log()` on a `fs-jetpack` object (with abs/rel path) crashes my app.

``` javascript
var jetpack = require('fs-jetpack')
console.log(jetpack.cwd('/Users/Shared/whatever'))
```

> A JavaScript error occurred in the main process
> Uncaught Exception:
> TypeError: Path must be a string. Received 2
>     at assertPath (path.js:7:11)
>     at Object.resolve (path.js:1148:7)
>     at resolvePath (/Users/Shared/xxx/build/node_modules/fs-jetpack/lib/jetpack.js:49:21)
>     at Object.inspect (/Users/Shared/xxx/build/node_modules/fs-jetpack/lib/jetpack.js:149:27)
>     at formatValue (util.js:348:21)
>     at inspect (util.js:198:10)
>     at exports.format (util.js:75:24)
>     at Console.log (console.js:43:37)
>     at App.<anonymous> (/Users/Shared/xxx/build/background.js:466:13)
>     at emitOne (events.js:101:20)

fs-jetpack 0.9.2

**Comment by szwacz** [comment]
This one wins the WTF of the year for me. As you can see in the call stack `console.log` calls node `util` which looks for `inspect` function on any object it's inspecting and guest what... jetpack has this function as part of its API.
I couldn't belive in what I'm seeing but this is part of the official node API: https://nodejs.org/api/util.html#util_util_inspect_object_options

Don't know how to resolve this yet. Filed issue: https://github.com/nodejs/node/issues/8071 to see what the almighty oracle will say.

**Comment by black-snow** [comment]
I knew you'd like this one ;)

As a workaround we could check `arguments.callee.caller` and `arguments.callee.caller.caller` and maybe some more (or all) levels up to see if `console.log` is part of the trace and if it is just return what `console.log` expects (or nothing or `toString`).

**Comment by black-snow** [comment]
This https://github.com/nodejs/node/pull/8013 doesn't help, does it? Only took a quick glimpse.

**Comment by szwacz** [comment]
Kind of it is. This is not general solution, but anyone can do `util.inspect.defaultOptions.customInspect = false` in their project and then this problem should stop appearing.

That beaing said the easiest solution i see to fix this is to change the `inspect` method name to something else (unfortunately).

**Comment by black-snow** [comment]
See https://github.com/nodejs/node/pull/8174

**Comment by szwacz** [comment]
Yep, shipped with node 6.6.0. Will see what we can do here.

**Comment by black-snow** [comment]
electron 1.4.0 ships w/ node 6.5.0, guess one of the next releases will be shipped w/ 6.6

Seems like all we have to do then is adding

```javascript
const util = require('util')

// exported object
{
    // other API stuff
    util.inspect.custom = depth => 'maybe something useful here'
}
```

to `jetpack.js`.

**Comment by szwacz** [comment]
Jetpack v0.10.2 fixes this for node versions 6.6.0 or newer. No way to fix this in nodes older than that.
